packages/orbit-components/src/Table/index.tsx

```diff
@@ -120,7 +120,7 @@ const Table = ({
   const handleScroll = () => {
     if (shadows && inner.current && table.current && outer.current) {
       setLeft(inner.current?.scrollLeft >= 5);
-      setRight(inner.current?.scrollLeft + outer.current?.clientWidth < table.current?.clientWidth);
+      setRight(inner.current.scrollLeft + outer.current.clientWidth < table.current?.clientWidth);
```

Comment:
Why can't we remove from line above and this one as well?

```suggestion
      setRight(inner.current.scrollLeft + outer.current.clientWidth < table.current.clientWidth);
```